packages/node_modules/@webex/plugin-meetings/src/config.js

```diff
@@ -16,6 +16,7 @@ export default {
     reconnection: {
       enabled: true,
       detection: true,
+      iceReconnectionTimeout: 10000,
```

Comment:
Could you add a quick comment saying what this does above it?

Reply:
Good idea! will add this comment.